modules/react-native-status/ios/RCTStatus/NetworkManager.m

```diff
@@ -0,0 +1,118 @@
+        configJSON:(NSString *)configJSON
+        callback:(RCTResponseSenderBlock)callback) {
+
+    NSData *configData = [configJSON dataUsingEncoding:NSUTF8StringEncoding];
+    NSError *error;
+    NSMutableDictionary *configDict = [NSJSONSerialization JSONObjectWithData:configData options:NSJSONReadingMutableContainers error:&error];
+    NSMutableDictionary *receiverConfig = configDict[@"receiverConfig"];
+    NSMutableDictionary *nodeConfig = receiverConfig[@"nodeConfig"];
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    NSURL *rootUrl =[[fileManager URLsForDirectory:NSLibraryDirectory inDomains:NSUserDomainMask] lastObject];
+    NSURL *multiaccountKeystoreDir = [rootUrl URLByAppendingPathComponent:@"keystore"];
+    NSString *keystoreDir = multiaccountKeystoreDir.path;
+    NSString *rootDataDir = rootUrl.path;
+
+    [receiverConfig setValue:keystoreDir forKey:@"keystorePath"];
+    [nodeConfig setValue:rootDataDir forKey:@"rootDataDir"];
+    NSString *modifiedConfigJSON = [Utils jsonStringWithPrettyPrint:NO fromDictionary:configDict];
+    NSString *result = StatusgoInputConnectionStringForBootstrapping(cs, modifiedConfigJSON);
+    callback(@[result]);
+}
+
+RCT_EXPORT_METHOD(sendTransactionWithSignature:(NSString *)txArgsJSON
+        signature:(NSString *)signature
+        callback:(RCTResponseSenderBlock)callback) {
+#if DEBUG
+    NSLog(@"sendTransactionWithSignature() method called");
+#endif
+    NSString *result = StatusgoSendTransactionWithSignature(txArgsJSON, signature);
+    callback(@[result]);
+}
+
+#pragma mark - SendTransaction
+
+RCT_EXPORT_METHOD(sendTransaction:(NSString *)txArgsJSON
+        password:(NSString *)password
+        callback:(RCTResponseSenderBlock)callback) {
+#if DEBUG
+    NSLog(@"SendTransaction() method called");
+#endif
+    NSString *result = StatusgoSendTransaction(txArgsJSON, password);
+    callback(@[result]);
+}
+
+RCT_EXPORT_METHOD(callRPC:(NSString *)payload
+        callback:(RCTResponseSenderBlock)callback) {
+    dispatch_async( dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        NSString *result = StatusgoCallRPC(payload);
+        dispatch_async(dispatch_get_main_queue(), ^{
+            callback(@[result]);
+        });
+    });
+}
+
+RCT_EXPORT_METHOD(callPrivateRPC:(NSString *)payload
+        callback:(RCTResponseSenderBlock)callback) {
+    dispatch_async( dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        NSString *result = StatusgoCallPrivateRPC(payload);
+        dispatch_async(dispatch_get_main_queue(), ^{
+            callback(@[result]);
+        });
+    });
+}
+
+#pragma mark - Recover
+
+RCT_EXPORT_METHOD(recover:(NSString *)message
+        callback:(RCTResponseSenderBlock)callback) {
+#if DEBUG
+    NSLog(@"Recover() method called");
+#endif
+    NSString *result = StatusgoRecover(message);
+    callback(@[result]);
+}
+
+@end
```

Comment:
Missing trailing newline character here and in a few other files.

Reply:
fixed and updated script to catch Objective-C files here : https://github.com/status-im/status-mobile/pull/18385/commits/5852012ba9c5e77ca8b9a85d5e36d7c216aeb31e